src/components/subscriptions/data/service.js

```diff
@@ -130,16 +150,11 @@ export function fetchSubscriptionUsers(options = {}) {
  * call to get this data.
  */
 export function sendLicenseReminder(options = {}) {
-  return Promise.resolve(options);
-}
+  const { userId, bulkRemind } = options;
+  const pendingSince = moment().format('MMM DD, YYYY');
```

Comment:
same comment here about not formatting here, but instead when reading this value within the component.